DEVELOPER.md

```diff
@@ -55,40 +90,65 @@ git remote add upstream https://github.com/MODFLOW-USGS/modflow6.git
 
 ## Building
 
-To build MODFLOW 6 run:
+You can build modflow with Visual Studio Code tasks as described [here](.vscode/README.md).
+For the more general instructions, continue to read this section.
+
+First configure the build directory:
 
 ```shell
-# Go to the pymake directory
-cd modflow6/pymake
+# bash (linux and macOS)
+meson builddir --prefix=$(pwd) --libdir=bin
 
-# run the modflow6 pymake build script
-python makebin.py -mc ../src/ ../bin/mf6
+# cmd (windows)
+meson builddir --prefix=%CD% --libdir=bin
 ```
 
-* Results are put in the bin folder.
+For the following compilations, only this command has to be executed:
+
+```shell
+meson compile -C builddir
+```
+
+In order to run the tests the binaries have to be installed by executing:
+
+```shell
+meson install -C builddir
+```
+
+The binaries can then be found in the `bin` folder.
+
 
 ## Running Tests Locally
 
-(Optional) For complete testing as done on Travis, clone the modflow6-testmodels repository:
+For complete testing as done on the CI, clone the modflow6-testmodels repository:
 
 ```shell
 # Clone your GitHub repository:
 git clone git@github.com:<github username>/modflow6-testmodels.git
 ```
 * The modflow6-testmodels repository must be cloned in the same directory that contains the modflow6 repository.
 
-To run tests:
+To run tests first change directory to the `autotest` folder:
 
 ```shell
-# Go to the autotest directory
 cd modflow6/autotest
+```
 
-# Run all modflow6 tests (including building executables and the mfio documentation - requires installation of LaTeX
-pytest -v
+Update your flopy installation by executing
 
-# Build MODFLOW 6, MODFLOW 6 utilities, and all versions of MODFLOW used in comparison tests
-pytest -v get_build_exes.py
+```shell
+python update_flopy.py
+```
 
+Then download latest release binaries and build latest release by executing:
```

Comment:
Testing may require other MODFLOW-related binary executables, distributed from https://github.com/MODFLOW-USGS/executables.  Testing also requires a binary executable of the last MODFLOW 6 officially released version, compiled in develop mode with the currently configured compiler.  To download MODFLOW-related binaries and to rebuild the last official MODFLOW 6 release, execute the following: